user-service/src/main/java/rs/edu/raf/banka1/model/BankAccount.java

```diff
@@ -7,19 +7,34 @@
 @Entity
 @Getter
 @Setter
-public class CurrentAccount {
+public class BankAccount {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+
+    @Enumerated(EnumType.STRING)
```

Comment:
jel ovo da bi lakse debugovao? Ako vec koristimo enum bolje da se cuva u bazi kao enum zbog ustede memorije?

Reply:
Sklonio sam to, trebalo bi da budem više eco-friendly sad.